setup.py

```diff
@@ -23,4 +23,7 @@
             "mu = mu.app.run",
         ],
     },
+    data_files = [('/etc/udev/rules.d', ['conf/90-usb-microbit.rule',]),
+                  ('/usr/share/pixmaps', ['conf/mu.png',]),
```

Comment:
All the applications I know, have their icons as normal files than links. That is why I did the copy.